docs/features-walk-through/banners.md

```diff
@@ -0,0 +1,40 @@
+# Site banners
+
+Banners can easily be added to your InvenioRDM instance through the admin panel. They can be added globally or to a specific path.
+
+## Adding a new banner
+
+**As an admin user** you can add new banners through the admin panel at `/administration`. This is also available through the user-menu in the top right corner of your instance:
+
+<img src="/features-walk-through/img/user-menu-admin.png" width="200"/>
+
+In the admin panel you will see "Banners" as a menu item in the sidebar. If you enter this, you will be shown an overview of all the banners added in your instance. In our case this list is empty, as we didn't add any banners yet.
+
+![Banners page](./img/empty_banners_page.png)
+
+To add a new banner, you can click the green "Create"-button in the top right corner. This will take you to the form where you can add all the details for your new banner:
+
+![New banner form](./img/new_banner_form.png)
+
+Here you can see that the first field "Start time" is already prefilled, however, you are free to update this to whatever start date and time you want.
+
+- "Start time" is when your banner will become visible on your site.
+- "End time" is when your banner will deactivate automatically.
+- "Message" is the content that will show in your banner.
+- "Url path" lets you specify where your banner should be visible. If left empty, the banner will show on every page of your site. If you specify a subpath, e.g. `/records`, the banner will show only on `/records` **and all its subpages**.
+- "Category" is where you can define the theme of your banner.
+- "Active" needs to be checked to activate the banner. This ensures that the banner will show on the given start time and deactivate on the given end time. If left unchecked, the start and end times will be ignored, and the banner will not be visible. This can be used to deactivate the banner temporarily, or permanently.
+
+In the following example, the banner will show up on all subpages of `/records` as an info-banner between April 3rd, 2023 at 12:00 and May 3rd, 2023 at 12:00.
+
+![Filled banner form](./img/filled_banner_form.png)
+
+When ready, click save and you will be taken back to the "Banners" page where the new banner will appear in the list:
+
+![Banner in list](./img/banner_added_list.png)
+
+Here you can see the details of your banner, and whether it is active or not. You can also edit the banner by clicking the "Edit" button in the "Actions" column, or delete it by clicking the trash can.
```

Comment:
Minor: the details of the banner can also be seen on the details page when you click on any banner id in the banners list.